docs/digger/installation/rhmap-properties.asciidoc

```diff
@@ -21,40 +21,37 @@ using Digger.
 
 | buildfarm_password
 | BUILDFARM_PASSWORD
-| The password of the user used to authenticate with the Jenkins instance.
+| Password for the username defined in `buildfarm_username`.
 
 | buildfarm_platforms
 | BUILDFARM_PLATFORMS
-| The mobile platforms that should be built using the Jenkins instance. Should
-be a comma separated list e.g. `android,ios`. Possible values are `ios` and
-`android`.
+| The mobile platforms to build using the Jenkins instance. Must be a comma separated list e.g. `android,ios`. Valid values: `ios`,`android`.
 
 | buildfarm_build_properties
 | BUILDFARM_BUILD_TIMEOUT
-| The number of seconds that RHMAP should wait before timing out a build.
+| The number of seconds that RHMAP will wait before timing out a build.
 
 | buildfarm_build_polling_interval
 | BUILDFARM_BUILD_POLLING_INTERVAL
 | The number of seconds between each poll of a Jenkins build.
 
 | buildfarm_csrf_enabled
 | BUILDFARM_CSRF_ENABLED
-| Whether CSRF protection is enabled on the Jenkins instance. It is recommended
-to enable CSRF protection.
+| CSRF protection. Recommended setting: Enabled
```

Comment:
are the optinos not `true,false`

Reply:
I have made a note of it with the other comments from this pr. will resolve in my pr